src/main/java/frc/robot/RobotContainer.java

```diff
@@ -36,7 +48,6 @@
   ResetOdometryCommand m_resetOdometryCommand = new ResetOdometryCommand(swerveDrivetrain);
```

Comment:
please change the names of `swerveDrivetrain` and `swerveJoystickCommand` so that it's immediately obvious that they are member variables.

Reply:
We should talk about this, but since we can't reuse commands, I'm against using m_ naming conventions. Reset odometry command is a unique case that we should consider if we want it to use m_